examples/chip-tool/commands/clusters/WriteAttributeCommand.h

```diff
@@ -244,14 +244,20 @@ class WriteAttribute : public InteractionModelWriter, public ModelCommand, publi
 
 private:
     // This constructor is private as it is not intended to be used from outside the class.
-    WriteAttribute(chip::ClusterId clusterId, chip::AttributeId attributeId, CredentialIssuerCommands * credsIssuerConfig) :
-        InteractionModelWriter(this), ModelCommand("write", credsIssuerConfig), mClusterIds(1, clusterId),
-        mAttributeIds(1, attributeId)
+    WriteAttribute(chip::ClusterId clusterId, chip::AttributeId attributeId, bool isWritable,
+                   CredentialIssuerCommands * credsIssuerConfig) :
+        InteractionModelWriter(this),
+        ModelCommand(isWritable ? "write" : "write-override", credsIssuerConfig), mClusterIds(1, clusterId),
```

Comment:
Perhaps:
```suggestion
        ModelCommand(isWritable ? "write" : "force-write", credsIssuerConfig), mClusterIds(1, clusterId),
```

Reply:
I thought of it too but I was trying to keep `write` as the first word. I agree `write-override` is not pretty.